esp-hal/src/usb_serial_jtag.rs

```diff
@@ -281,7 +281,7 @@ where
     /// number of bytes in the FIFO is larger than `buf`.
     pub fn drain_rx_fifo(&mut self, buf: &mut [u8]) -> usize {
```

Comment:
I wonder if we should do the same as in https://github.com/esp-rs/esp-hal/pull/2895